Cargo.toml

```diff
@@ -72,7 +72,7 @@ openssl = "0.10"
 p256 = { version = "0.13.2", default-features = false, features = ["alloc", "ecdsa", "pkcs8"] }
 pkcs8 = "0.10.2"
 pki-types = { package = "rustls-pki-types", version = "1.10", features = ["alloc"] }
-rand_core = { version = "0.6", features = ["getrandom"] }
+rand_core = { version = "0.9", features = ["getrandom"] }
```

Comment:
```suggestion
rand_core = { version = "0.9", features = ["os_rng"] }
```

Reply:
This works for me, with some updates & workarounds: https://github.com/brody4hire/rustls/pull/5

Maybe better if this update can be done in `x25519-dalek` & `signature` crates first.